lib/ecto/adapters/postgres.ex

```diff
@@ -119,4 +119,10 @@ defmodule Ecto.Adapters.Postgres do
     args = args ++ ["--quiet", "--host", database[:hostname], "-d", "template1", "-c", sql_command]
     System.cmd("psql", args, env: env, stderr_to_stdout: true)
   end
+
+  @doc false
+
+  def supports_ddl_transaction? do
```

Comment:
@josevalim I just moved the `supports_ddl_transaction?` function to the `Ecto.Adapters.Postgres` module. The problem now is that we are defining the behaviour inside the `Ecto.Adapter.Migrations` module and it will warn us that it is not implemented. Shall we move this behaviour definition to another place?